server/config/util.go

```diff
@@ -111,3 +111,44 @@ func parseUrls(s string) ([]url.URL, error) {
 
 	return urls, nil
 }
+
+func parseMode(modes string) ([]ServiceMode, error) {
```

Comment:
Need unit test to show the valid and invalid input string of 'modes'.

Reply:
yes, I will add them later.